payroll-processor-functions/src/payroll-processor-functions/Infrastructure/EnvironmentSettings.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using LanguageExt;
+
+namespace PayrollProcessor.Functions.Infrastructure
+{
+    public static class EnvironmentSettings
```

Comment:
This is a wrapper around configuration access that uses `Option<T>` to force callers to do something if the value isn't found (eg throw an exception)